src/crimson/admin/admin_socket.cc

```diff
@@ -0,0 +1,696 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "common/version.h"
+#include "crimson/net/Socket.h"
+#include "crimson/admin/admin_socket.h"
+#include "crimson/common/log.h"
+#include "seastar/net/api.hh"
+#include "seastar/net/inet_address.hh"
+#include "seastar/core/reactor.hh"
+#include "seastar/core/thread.hh"
+#include "seastar/util/std-compat.hh"
+#include <boost/algorithm/string.hpp>
+
+/**
```

Comment:
i'd suggest just use "//" for single line and for multiple line comments in crimson.